examples/quine.html

```diff
@@ -3,10 +3,13 @@
 <head>
 <meta charset="utf-8" />
 <title>Making Quines Prettier</title>
-<!-- The defer is not necessary for autoloading, but is necessary for the
-     script at the bottom to work as a Quine. -->
-<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer="defer"></script>
-<style>.operative { font-weight: bold; border:1px solid yellow }</style>
+<!-- The defer attribute is not necessary for autoloading, but is necessary
+     for the script at the bottom to work as a Quine. -->
+<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer></script>
```

Comment:
You really shouldn't be using `cdn.rawgit.com` with a branch name as the specifier. It will only cache that path once, which means it cached an older version of the branch and won't update.

_Yes, technically this is no longer true for the master branch. But until a few days ago, there were no exceptions, and this example is quite old._